src/metatrain/experimental/mace/documentation.py

```diff
@@ -0,0 +1,203 @@
+"""
```

Comment:
I would ad a comment in the first line saying that this is a generated file. Do not edit. Rather edit the source file at XXX

Reply:
Actually I don't know what to do with this :sweat_smile: Right now the generating script just generates a documentation file so that you can take whatever you want from it, it doesn't replace the real `documentation.py`. This is because I think it could make sense to document better the hypers. MACE's documentation strings are really poor...